lib/v0_bindings_extensions.rb

```diff
@@ -83,11 +83,21 @@ def self.create_from_query_result(query_result)
 
   Api::V0::Bindings::GetHighlightsParameters.class_exec do
     def query(user_id:)
-      highlights = ::Highlight.by_user(user_id)
+      users = []
+
+      if sets.blank? or sets.include?('user:me')
+        users.push(user_id)
+      end
+      if  sets.to_a.include?('curated:openstax') and scope_id.present?
```

Comment:
1. Extra space after `if`
2. I think `sets` is already an array, so I don't think you need `.to_a`

Reply:
the to_a is a null check, i can change it to be more explicit